addons/product/models/product_template.py

```diff
@@ -3,6 +3,7 @@
 import itertools
 import logging
 from collections import defaultdict
+import copy
```

Comment:
this import should be before the `itertools` import to keep the imports ordered.